near/src/chain_rollback.rs

```diff
@@ -0,0 +1,18 @@
+use std::path::Path;
```

Comment:
I was actually thinking to do this as separate tool (e.g. like test-utils/state-viewer), just because don't think this should exposed by default